modules/ROOT/pages/persistence-gateway.adoc

```diff
@@ -0,0 +1,147 @@
+
+== Persistent Gateway Limits
+
+The following table lists the limits on the data stored by the Persistence Gateway:
+
+[%header,cols="3*a"]
+|===
+| Limit | Description | Default Value
+| Maximum TTL. | The amount of time data is stored in the Persistence Gateway  | 30 days
+| Maximum key length | | 256 characters
+| Maximum value length | | 10 MB
+| Maximum partitions per application | | 10
+| Maximum keys per partition | | 1000
+|===
+
+
+[Configure-Persistence-Gateway]
+== Configure Persistence Gateway
+
+To enable Persistence Gateway, you must create a Kubernetes custom resource that allows the cluster to connect to your persistence data store.
+
+=== Prerequisties
+
+Before enabling Persistence Gateway, ensure that you have created a database to serve as the data source for data stored by the Persistence Gateway. This database must be compatible with PostgresSQL 9 and above. 
+
+=== Create a Kubernetes Secret and Kubernetes Custom Resource
+
+. Create a Kubernetes secret:
+.. Copy the Kubernetes secret template from <<k8-secret-template>> to a file called `kubernetes-secret.yaml`.
+.. Create a base64-encoded connection string. 
++
+The standard URI for this string is `postgres://username:pass@host:port/databasename`. However, the requirements for your database may be different.
+.. Copy the encoded string to the `persistence-gateway-creds:` field of `kubernetes-secret.yaml`. 
+.. Run `kubectl apply -f kubernetes-secret.yaml`.
+
+. Create a custom resource for your data store:
+.. Copy the custom resource template from <<k8-custom-resource-template>> to a file called `custom-resource.yaml`.
+.. Ensure the value of `secretRef: name` matches the `name` field defined in your Kubernetes secret file.
+.. Modify other fields of the custome resource template as required for your environment.
+.. Run `kubectl apply -f custom-resource.yaml`.
+
+. Check the log of the Persistence Gateway pod to ensure it can communicate with the database:
++
+----
+kubectl logs -f persistence-gateway-6dfb98949c-7xns9 -nrtf
+----
++
+The output of this command should be similar to the following:
++
+----
+2021/04/09 16:35:31 Connecting to PostgreSQL backend...
+2021/04/09 16:35:32 Starting watcher for /var/run/secrets/rtf-object-store/persistence-gateway-creds
+2021/04/09 16:35:32 Watching for changes on /var/run/secrets/rtf-object-store/persistence-gateway-creds
+2021/04/09 16:35:35 Succesfully connected to the PostgreSQL backend.
+192.168.2.101 - - [09/Apr/2021:16:35:55 +0000] "GET /api/v1/status/ready HTTP/1.1" 200 2 "" "kube-probe/1.18+"
+----
+
+[[k8-secret-template]]
+=== Kubernetes Secret Template
+
+Use the example template below to create a Kubernetes secret. The Kubernetes cluster uses this secret to connect to your data store.
+
+----
+apiVersion: v1
+data:
+    persistence-gateway-creds: cG9zdGdyZXNxbDovL3Bvc3RncmVzOlRlc3RpbmdQb3N0Z3Jlc0BhdXRvbWF0aW9uLXRlc3RpbmctcnYuY2ZqcHFoeTZlYWhzLnVzLWVhc3QtMS5yZHMuYW1hem9uYXdzLmNvbTo1NDMyL1JURg==
```

Comment:
@hegderohit89 Can you update the secret? The above one is referencing to automation postgresql string: `postgresql://postgres:TestingPostgres@automation-testing-rv.cfjpqhy6eahs.us-east-1.rds.amazonaws.com:5432/RTF`

Reply:
fixed